indexer/packages/postgres/src/models/compliance-data-model.ts

```diff
@@ -0,0 +1,44 @@
+import { Model } from 'objection';
+
+import { NumericPattern } from '../lib/validators';
+import { IsoString } from '../types';
+
+export default class ComplianceDataModel extends Model {
+  static get tableName() {
+    return 'compliance_data';
+  }
+
+  static get idColumn() {
+    return 'address';
+  }
+
+  static relationMappings = {};
+
+  static get jsonSchema() {
+    return {
+      type: 'object',
+      required: [
+        'address',
+        'sanctioned',
+        'updatedAt',
+      ],
+      properties: {
+        address: { type: 'string' },
+        chain: { type: ['string', 'null'], default: null },
```

Comment:
so is chain either `dydx` or null if it's for all other chains?

Reply:
No, it can be `eth` or names of any other chain.